templates/mixins/lang-router.pug

```diff
@@ -1,3 +1,5 @@
 mixin buildRoute(routeName, params, text)
 	a(href=url(getLocale() + '.' + routeName, params))= text
 
+mixin buildRouteToId(routeName, params, id, text)
+	a(href=url(getLocale() + '.' + routeName, params)+'#'+id)= text
```

Comment:
Ha! Creative way to get around that massive header... I think we might want to look into removing the hero header on post pages so that a redirect doesn't have to do this but I love this little hack :)

Reply:
@iamandrea how's your busyness level these days? :)

Reply:
Not sure what you mean by post pages, but here the idea was to enable links to paragraphs. That is why I just included the id tag.